src/nvim/ui.c

```diff
@@ -142,6 +142,104 @@ void ui_event(char *name, Array args)
   }
 }
 
+bool attr2hlattr(int attr_code, bool use_rgb, HlAttrs *out)
```

Comment:
sorry, I should have clarified. These functions should live in syntax.c (wasn't that where they originally were?)